lib/internal/streams/utils.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 
 const {
-  Symbol,
+  SymbolFor,
   SymbolAsyncIterator,
   SymbolIterator,
 } = primordials;
 
-const kDestroyed = Symbol('kDestroyed');
-const kIsErrored = Symbol('kIsErrored');
-const kIsReadable = Symbol('kIsReadable');
-const kIsDisturbed = Symbol('kIsDisturbed');
+const kDestroyed = SymbolFor('nodejs.stream.destroyed');
+const kIsErrored = SymbolFor('nodejs.stream.errored');
+const kIsReadable = SymbolFor('nodejs.stream.readable');
+const kIsWritable = SymbolFor('nodejs.stream.writable');
+const kIsDisturbed = SymbolFor('nodejs.stream.disturbed');
```

Comment:
This was discussed again today in the TSC meeting, turns out that having those a public documented properties would probably do more harm than good if they start being used in the wild. In order to make it very clear that it should never be used unless you know what you're doing, we could either:

- expose them as `Stream._internalSymbols` or similar.
- use `Symbol.for('_nodejs...')` instead of `Symbol.for('nodejs...')` (or some other variant that makes it very clear that's internal `nodejs.INTERNAL_DO_NOT_USE_OR_YOULL_BE_FIRED` would work for me).

wdyt?